src/components/TodoForm.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./Todo.css";
+
+function TodoForm(props) {
+  return (
+    <div className="form">
+      <form onSubmit={props.formSubmitHandler}>
+        <input
+          type="text"
+          value={props.todoTask}
+          className="todo-input"
+          onChange={props.handleChange}
```

Comment:
3. `name="newTask"` is what we need for `[e.target.value]` to work
```suggestion
          onChange={props.handleChange}
          name="newTask"
```